src/components/Product/ProductSearchBar.tsx

```diff
@@ -41,7 +40,9 @@ const TextElement = styled.div`
   }
 `;
 
-const ProductSearchBar = () => {
+const ProductSearchBar = (props: ProductSearchBarProps) => {
```

Comment:
if this component no longer does any search functionality, change to ProductSortBar

Reply:
Changed